src/main/java/de/chojo/repbot/commands/repadmin/RepAdmin.java

```diff
@@ -30,25 +30,40 @@ public RepAdmin(Guilds guilds, Configuration configuration, RoleAssigner roleAss
                                 .argument(Argument.user("user", "command.repadmin.reputation.add.user.description")
                                                   .asRequired())
                                 .argument(Argument.integer("amount", "command.repadmin.reputation.add.amount.description")
+                                                  .min(1)
+                                                  .max(1_000_000)
                                                   .asRequired()))
                         .subCommand(SubCommand.of("remove", "command.repadmin.reputation.remove.description")
                                 .handler(new Remove(roleAssigner, guilds))
                                 .argument(Argument.user("user", "command.repadmin.reputation.remove.user.description")
                                                   .asRequired())
                                 .argument(Argument.integer("amount", "command.repadmin.reputation.remove.amount.description")
+                                                  .min(1)
+                                                  .max(1_000_000)
                                                   .asRequired()))
                         .subCommand(SubCommand.of("set", "command.repadmin.reputation.set.description")
                                 .handler(new Set(roleAssigner, guilds))
                                 .argument(Argument.user("user", "command.repadmin.reputation.set.user.description")
                                                   .asRequired())
                                 .argument(Argument.integer("amount", "command.repadmin.reputation.set.amount.description")
+                                                  .min(0)
+                                                  .max(1_000_000)
                                                   .asRequired())))
                 .group(Group.of("resetdate", "command.repadmin.resetdate.description")
                         .subCommand(SubCommand.of("set", "command.repadmin.resetdate.set.description")
                                 .handler(new SetResetDate(guilds))
-                                .argument(Argument.integer("year", "command.repadmin.resetdate.set.year.description").asRequired().min(2016).max(LocalDate.now().getYear()))
-                                .argument(Argument.integer("month", "command.repadmin.resetdate.set.month.description").asRequired().min(1).max(12))
-                                .argument(Argument.integer("day", "command.repadmin.resetdate.set.day.description").asRequired().min(1).max(31)))
+                                .argument(Argument.integer("year", "command.repadmin.resetdate.set.year.description")
+                                                  .asRequired()
+                                                  .min(2016)
+                                                  .max(LocalDate.now().getYear()))
```

Comment:
What about the case when I would like to set the reset date to `01.01.2023` on `31.12.2022`? I know this is a very specific case, so feel free to ignore it.

Reply:
> What about the case when I would like to set the reset date to `01.01.2023` on `31.12.2022`? I know this is a very specific case, so feel free to ignore it.

Mhmm yeah we should probably allow to set one year in the future